src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -88,6 +88,34 @@ public static void EnsureDirectoryExists(string directoryPath)
             }
         }
 
+        public static bool TryEnsureDirectoryExists(string directoryPath)
+        {
+            try
+            {
+                EnsureDirectoryExists(directoryPath);
+
+                return true;
+            }
+            catch
```

Comment:
What exception are we expecting here? Since `EnsureDirectoryExists` checks for directory existence before creating it, the kinds of exceptions I'd expect to see here probably shouldn't be swallowed, e.g. permissions problems (resulting in a nonexistent directory but returning `true`) or invalid values for `directoryPath`.